tests/foreman/ui_airgun/test_errata.py

```diff
@@ -0,0 +1,107 @@
+"""UI Tests for the errata management feature
+
+:Requirement: Errata
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from nailgun import entities
+
+from robottelo.cli.factory import (
+    setup_org_for_a_custom_repo,
+    setup_org_for_a_rh_repo,
+)
+from robottelo.constants import (
+    DISTRO_RHEL7,
+    FAKE_1_CUSTOM_PACKAGE,
+    FAKE_2_ERRATA_ID,
+    FAKE_6_YUM_REPO,
+    PRDS,
+    REPOS,
+    REPOSET,
+)
+from robottelo.decorators import tier3
+from robottelo.vm import VirtualMachine
+
+CUSTOM_REPO_URL = FAKE_6_YUM_REPO
+CUSTOM_REPO_ERRATA_ID = FAKE_2_ERRATA_ID
+
+
+@tier3
+def test_end_to_end(session):
```

Comment:
what's about `content_hosts` and `repositories` tabs. Can you test anything from them to have more real `end-to-end`?

Reply:
Errata installation goes through content_hosts tab, searches for host, selects it etc., so that tab is implicitly covered, i'm not sure it's worth doing the same actions once more just to be more explicit.
Will think about repositories tab coverage, thanks for suggestion.